apps/condo/pages/billing/index.tsx

```diff
@@ -4,14 +4,16 @@ import React, { useCallback } from 'react'
 import { useIntl } from '@open-condo/next/intl'
 import { useOrganization } from '@open-condo/next/organization'
 
+import { CONTEXT_FINISHED_STATUS, CONTEXT_VERIFICATION_STATUS } from '@condo/domains/acquiring/constants/context'
 import { AcquiringIntegrationContext as AcquiringContext } from '@condo/domains/acquiring/utils/clientSchema'
 import { BillingPageContent } from '@condo/domains/billing/components/BillingPageContent'
 import { BillingAndAcquiringContext } from '@condo/domains/billing/components/BillingPageContent/ContextProvider'
 import { BillingOnboardingPage } from '@condo/domains/billing/components/OnBoarding'
 import { BillingIntegrationOrganizationContext as BillingContext } from '@condo/domains/billing/utils/clientSchema'
 import LoadingOrErrorPage from '@condo/domains/common/components/containers/LoadingOrErrorPage'
-import { CONTEXT_FINISHED_STATUS } from '@condo/domains/miniapp/constants'
+import { CONTEXT_FINISHED_STATUS as BILLING_FINISHED_STATUS } from '@condo/domains/miniapp/constants'
```

Comment:
IMO it will be difficult to determine constant usages at the code below

Reply:
This is due to the fact that prior to this, all integration lived as miniaps and had the same types of statuses. Now billing and acquiring have separated and acquiring required an additional status. So I moved them to separate constants. I did not move billing, because in general it still works in a similar way to mini-apps. 

If you know a more elegant solution to this problem, I'm willing to add it.

(I also didn't want to do something like `[...MINIAPP_STATUSES, NEW_STATUS]` because of some extra logic on acquiring statuses)

Reply:
It seems worth considering moving these constants to the current domain. Anyway it's an optional, so you can leave it as is